client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/RestService.java

```diff
@@ -679,8 +679,8 @@ public Config deleteSubjectConfig(String subject)
 
   public Config deleteSubjectConfig(Map<String, String> requestProperties, String subject)
```

Comment:
Let's change this method name to `deleteConfig` now.  It should be fine as this is an internal method.  There's an overloaded method that should be changed too.